src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -38,6 +37,18 @@ import BulkSelect from '@redhat-cloud-services/frontend-components/BulkSelect';
 import axios from 'axios';
 import { Stack, StackItem } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
 
+interface Bundle {
+  id: string;
+  name: string;
+  displayName: string;
+  children: null | Bundle[];
```

Comment:
I think children can be optional `children?: Bundle[]`. Or is there is a specific need for the `null` value?

Reply:
Good catch! I think you're right.